openfisca_us/variables/gov/states/ca/tax/income/dedutions/standard/ca_standard_deduction.py

```diff
@@ -0,0 +1,18 @@
+from openfisca_us.model_api import *
+
+
+class ca_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "California Standard Deduction"
+    unit = USD
+    documentation = "https://www.ftb.ca.gov/forms/2021/2021-540.pdf"
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ca.tax.income.deductions.standard
+        filing_status = tax_unit("filing_status", period)
+        amount_if_eligible = p.amount[filing_status]
+        in_ca = tax_unit.household("state_code_str", period) == "CA"
```

Comment:
We now actually have an inbuilt way of restricting to CA tax units that optimises performance - instead of this line, we'll just need to add at the top:

```python
definition_period = YEAR
defined_for = StateCode.CA
```

Reply:
Done